test/karma/test-invisible-prehydration/stencil.config.ts

```diff
@@ -0,0 +1,21 @@
+import { Config } from '../../../dist/declarations';
+
+export const config: Config = {
+  namespace: 'TestPrehydratedDefaultStyles',
+  tsconfig: 'tsconfig.json',
+  outputTargets: [
+    {
+      type: 'www',
+      dir: '../www',
+      empty: false,
+      indexHtml: 'prehydrated-styles.html',
+      serviceWorker: null,
+    },
+    {
+      type: 'www',
+      empty: false,
+      indexHtml: 'prehydrated-styles.html',
+      serviceWorker: null,
```

Comment:
I'm not sure I fully understand why we need both of these output targets - would you mind explaining the usage of each to me?

Reply:
My intent here was to allow for local debugging if necessary, and for placing the build content within the testable directory. It may seem clunky, and it is, but it's not as bad as it seems since this is only for putting the built files where the tests were already running.

Reply:
I've got a good amount of brain fog ATM - but don't we have that ability (local debugging) from running `npm start` in the karma directory?

Reply:
Hmmm so it looks like we don't serve the HTML in the local dev server, just the minified JS. Maybe that's what you were getting at?
![Screen Shot 2021-08-25 at 8 56 44 AM](https://user-images.githubusercontent.com/1930213/130794523-2c5c4763-cef5-4cf6-b183-3c3f1687dccc.png)